dev-cpp/pystring/metadata.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<!-- maintainer-needed -->
```

Comment:
Add yourself as maintainer? We want to avoid adding new m-n packages

Reply:
Ping. I'll add you for now and can remove if you're unhappy?

Reply:
Sure!
I sadly didn't have any time to work on this today.

However if you feel that you want this merged ASAP you can add me as maintainer for the packages in this pull request.

Don't forget to adress waebbl's feedback on the embree package as well.

Reply:
Thank you! I'll get on that :)